lib/mspec/runner/actions/constants_leak_checker.rb

```diff
@@ -0,0 +1,86 @@
+class ConstantsLeakChecker
+  attr_accessor :constants_new
+
+  def initialize
+    @constants_before = Module.constants
```

Comment:
I would use the clearer `Object.constants(true)` rather than `Module.constants`.